cli/tests/test_support.bash

```diff
@@ -176,7 +176,12 @@ teardown_file() { common_file_teardown; }
 common_test_teardown() {
   # wait for any running klaus proceses to finish
   if [[ -n "${FLOX_DATA_DIR:-}" ]]; then
-    if pids="$(pgrep -f klaus.*${FLOX_DATA_DIR?})"; then
+    # This is a hack to essentially do a `pgrep` without having access to `pgrep`.
+    # The `ps` prints `<pid> <cmd>`, then we use two separate `grep`s so that the
+    # grep command itself doesn't get listed when we search for the data dir.
+    # The `cut` just extracts the PID.
+    local pids
+    if pids="$(ps -eo pid,args | grep klaus | grep ${FLOX_DATA_DIR?} | cut -d' ' -f1)"; then
```

Comment:
nonblocking: if you use `ps -o comm | grep klaus` you won't get `grep klaus included`

Reply:
We also won't get the arguments though, and we need those for finding the data directory path